petab/visualize/helper_functions.py

```diff
@@ -259,6 +259,8 @@ def create_figure(uni_plot_ids):
 
     # initialize figure
     fig, ax = plt.subplots(int(num_row), int(num_col), squeeze=False)
+    for axes in ax.flat[num_subplot:]:  # trim subplots output to the correct size
```

Comment:
line is too long >79, just put comment above. Travis check should pass after fixing this.